### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by areski** [Feverup/django-crispy-admin#1]
The link to documentation is not working

**Comment by iknite** [comment]
Yeah, sadly It will never goes live, We're deprecating this repository since we're not going to use it
